Sources/SyndiKit/Formats/OPML/OPMLBody.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+public extension OPMLSubscription {
+  struct Body: Codable, Equatable {
+
+    public struct Outline: Codable, Equatable {
```

Comment:
move this to inside `OPML` instead of a _grandchild_ and put it in a separate file